api/proto/auth/auth.proto

```diff
@@ -37,7 +37,7 @@ message Event {
         services.CertAuthorityV2 CertAuthority = 3
             [ (gogoproto.jsontag) = "cert_authority,omitempty" ];
         // StaticTokens is filled in static-tokens related events
-        services.StaticTokensV2 StaticTokens = 4
+        services.StaticTokensV2 StaticTokens = 4 
```

Comment:
Nit: Extra trailing whitespace here and below. You can configure your IDE/text editor to auto-trim these for you.